playbooks/robusta_playbooks/job_restart_on_oomkilled.py

```diff
@@ -0,0 +1,247 @@
+from robusta.api import (
+    action,
+    ActionParams,
+    Optional,
+    JobEvent,
+    Finding,
+    Container,
+    FindingSource,
+    logging,
+    FindingType,
+    MarkdownBlock,
+    ResourceRequirements,
+    RobustaJob,
+    ObjectMeta,
+    JobSpec,
+    PodSpec,
+    PodTemplateSpec,
+)
+from ...src.robusta.core.playbooks.job_utils import get_job_latest_pod
+import bitmath as bitmath
+
+
+class IncreaseResources(ActionParams):
+
+    """
+    :var increase_by: (optional).Users will specify how much they want to increase in each restart, (e.g: 500MiB).
+    :var max_resource: This variable prevent an infinite loop of job's pod crashing and getting more memory.The action won't increase the memory again when the "Max" limit reached, (eg: 1GiB).
+    """
+
+    increase_by: Optional[str] = 0
```

Comment:
can we set it to 500Mi by default? Otherwise if it is 0 it won't do anything, right? it may also get into an infinite loop

Reply:
added.